sdk/js/src/EventEmitter.ts

```diff
@@ -23,7 +24,7 @@ export class EventEmitter {
         checkParamType('key', key, 'string')
         checkParamType('handler', handler, 'function')
 
-        const eventNames = Object.keys(EventNames).map((e) => e.toLowerCase())
+        const eventNames = Object.values(EventNames)
         if (!eventNames.includes(key) &&
```

Comment:
I'm pretty sure this was a bug - using the EventNames keys instead of values meant that, for ex, `variable_updated` was a valid event to subscribe to, but it never receives any events